src/OrbitGl/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ target_sources(
          MajorPageFaultsTrack.h
          MemoryTrack.h
          MinorPageFaultsTrack.h
+         MockBatcher.h
```

Comment:
This is for testing only, right? So it shouldn't go into the library, only into the testing target

Reply:
Yes, thanks!